src/check/test/type_checking_integration.zig

```diff
@@ -282,14 +282,14 @@ test "check type - def - forward ref" {
 }
 
 test "check type - def - nested lambda with wrong annotation" {
-    return error.SkipZigTest;
+    if (true) return error.SkipZigTest;
```

Comment:
Oh that `if (true)` is a cool trick to avoid unreachable warnings after a skip! I never thought of that. 😃